cdap-data-fabric/src/main/java/co/cask/cdap/data2/metadata/lineage/field/ProgramRunOperations.java

```diff
@@ -26,20 +26,14 @@
  * Represents a set of program runs that performed the same set of field operations.
  */
 public class ProgramRunOperations {
-  private final long checksum;
   private final Set<ProgramRunId> programRunIds;
   private final Set<Operation> operations;
 
-  public ProgramRunOperations(long checksum, Set<ProgramRunId> programRunIds, Set<Operation> operations) {
-    this.checksum = checksum;
+  public ProgramRunOperations(Set<ProgramRunId> programRunIds, Set<Operation> operations) {
```

Comment:
Missed this change in my last commit. Removed checksum from ProgramRunOperations, since its not needed.